target/linux/airoha/files/drivers/pinctrl/pinctrl-airoha.c

```diff
@@ -15,11 +17,16 @@
 #include <linux/pinctrl/pinconf-generic.h>
 #include <linux/pinctrl/pinmux.h>
 #include <linux/platform_device.h>
+#include <linux/of_irq.h>
```

Comment:
@merbanan can you please put the include in alphabetic order?

Reply:
Ack